src/main/java/com/pokegoapi/api/map/Map.java

```diff
@@ -91,8 +95,8 @@ public void clearCache() {
 	 *
 	 * @return a List of CatchablePokemon at your current location
 	 */
-	public List<CatchablePokemon> getCatchablePokemon() throws LoginFailedException, RemoteServerException {
-		List<CatchablePokemon> catchablePokemons = new ArrayList<>();
+	public Set<CatchablePokemon> getCatchablePokemon() throws LoginFailedException, RemoteServerException {
```

Comment:
You can't change the response type here. That would affect anyone/anything that is currently using this method.